sphinx/tutorial/cylc/runtime/configuration-consolidation/index.rst

```diff
@@ -68,7 +68,7 @@ Consolidating Configuration
           [[dependencies]]
               [[[T00/PT3H]]]
                   graph = """
-                      get_observations_belmullet => consolidate_observations
+                      get_observations_aldergrove => consolidate_observations
                       get_observations_camborne => consolidate_observations
                       get_observations_heathrow => consolidate_observations
                       get_observations_shetland => consolidate_observations
```

Comment:
FYI: The original list of sites definitely weren't selected solely on the basis of line length.

Reply:
You're lucky I didn't try to sneak 03257 (RAF Leeming) in there.